_docs/style/colors.md

```diff
@@ -3,28 +3,160 @@ title: Color Palette
+
 resource: true
 status: complete
 ---
 
 {% include toc.html %}
 
 <section class="static-section"  markdown="1">
-### Primary Colors
 
-Primary colors should be used sparingly for primary and secondary actions,
-alerts, or data visualizations.
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## The Helix color palette
+
+The Helix color palette consists of deep modern hues juxtaposed with dichromatic steel and gray families. Our color palette has been selected to enhance usability while adhering to accessibility criteria. The guidelines outline color usage for individual components.
+
+{% figure [caption:"Helix Color Palette"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/color-palette/color-palette.png" width="1900"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Guidelines
+
+We have designed the Helix color palette intentionally so that it:
+
+- **Communicates meaning**: We use color to communicate meaning so that the user can make sense of all of the data presented to them.
+- **Draws the attention of the user**: We use color to the draw user's attention to the content that is most requires attention and away from areas that require less attention.
+- **Orients the user**: We use color to enhance way-finding so that users don't become confused about location.
+
+Any deviation from the predefined colors introduces an inconsistency that can be confusing to the user. Strict adherence to the color palette and its families is important so that we can successfully communicate meaning, draw attention, and orient the user.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you work with the color palette:
+
+- Do not add colors to the color palette or the color families.
+- Be intentional about using color families. Don't use colors outside of a color family for a component that belongs to the family.
+- When you apply a color to an object, consider all of the other colors used on the page. Because color commands the attention of the user, ensure that the colors you use get the right amount of attention, regardless of the data set, device type, or permission setting.
+- Ensure that you construct interactive states for interactive objects.
+- Use all of the interactive states for an interactive object. Do not use just some of the interactive states.
```

Comment:
we don't really need the second sentence do we?

Reply:
Done.